index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body class="container">
+    <header class="toolbar">
+        <img src="./img/my-logo.png" alt="my logo" class="my-logo">
+        <button class="hamburger-menu"><img src="./img/hamburger.png" alt="Menu" id="hamburger-img"></button>
+    </header>
+    <section class="headline">
+        <h3>I'm Newton <br>Glad to see you!</h3>
```

Comment:
- [ ] Please make sure that the main title (e.g., "I'm Marianne ...") is an `h1` tag.
  * If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.